css/index.css

```diff
@@ -143,4 +216,100 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+.box:first-child {
+    background: teal;
+}
+
+.box:nth-child(2) {
```

Comment:
Using `nth-child()` for this project is totally OK. However, I'd be cautious about using this because it's actually quite easy to change the order of your items and `nth-child()` styles based on order. So you might want to change the order of your boxes, but run into an issue where your backgrounds stay in the same order regardless of what order you move your boxes into. Using ids, or classes, can prevent that. 

Just food for thought!